src/pydvl/influence/frameworks/torch_differentiable.py

```diff
@@ -261,6 +283,19 @@ def cat(a: Sequence[torch.Tensor], **kwargs) -> torch.Tensor:
     return torch.cat(a, **kwargs)
 
 
+def zero_tensor(
+    shape: Sequence[int], dtype: Union[np.dtype, torch.dtype], **kwargs
+) -> torch.Tensor:
+    """Returns a tensor of shape :attr:`shape` filled with zeros."""
+    if isinstance(dtype, np.dtype):
+        dtype = TORCH_DTYPES[str(dtype)]
```

Comment:
Perhaps this would be more general and work better:

```suggestion
        dtype = getattr(torch, dtype.name)
```

I found it in [this issue](https://github.com/pytorch/pytorch/issues/40568#issuecomment-650198299)